src/logger/HttpClientLogger_test.go

```diff
@@ -0,0 +1,73 @@
+package logger
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+)
+
+//Testing HTTP Logger
+
+func TestCreateInstance(t *testing.T) {
+
+	//Creating a single instance
+	logger := newLogger()
+
+	assert.NotNil(t, logger)
+	assert.Equal(t, HttpLogger.AGENT, logger.AGENT)
```

Comment:
I guess I should have communicated more, but the baselogger has updated syntax to reflect go conventions, so logger.AGENT becomes logger.Agent() (getter is Agent() and the actual non exported value is just agent) same issues with all the other functions that assume the Java naming. Should be an easy fix we can do later today.